scss/tech.scss

```
@@ -3,36 +3,46 @@
 }
 
 .image-icon {
+  align-self: center;
   height: $tech-logo-size;
-  width: auto;
+  justify-content: center;
+  max-height: 50%;
+  max-width: 50%;
 }
 
 .tech {
   background-color: $green;
   border-radius: 10px;
   display: flex;
   flex-direction: column; 
-  margin: 30px;
+  margin: 15px;
   padding-bottom: 30px;
   padding-top: 10px;
   position: relative;
   text-align: center; 
-  width: 230px;
+  width: 250px;
+}
 
-  &::before {
-    border: 2px solid $green;
-    border-radius: 10px;
-    content: "";
-    height: 100%;
-    left: -10px;
-    position: absolute;
-    top: -10px;
-    width: 100%;
-    z-index: -1;  
-  }    
+.tech::before {
+  border: 2px solid $green;
+  border-radius: 10px;
+  content: "";
+  height: 100%;
+  left: -10px;
+  position: absolute;
+  top: -10px;
+  width: 100%;
+  z-index: -1; 
+}
+  
+.technologies {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin: 20px 200px 20px 120px;
```

I recommend you change this by: 
```suggestion
  margin: 20px auto 120px;
```
And add `max-width: 100%` and `width: 1000px` (`1000px` is a random value replace it for another value)